### PR TITLE
Wordsmithing, making dir names consistent, implemented RSTs, implemented Cards

### PR DESCRIPTION
**Pull request by jrhoun**
Hi @rbohl !

This is almost ready to merge - mostly just needs either:

a. added screenshots for places that I put a comment
b. updated screenshots to reflect the app that is being built in the "creating a low code application" article.

Once you send that to me, I consider it good enough to push up.

One area that I still feel uneasy about is in the introduction - and that's because I'm not sure if we're doing a good enough job at presenting the case for using App Builder and how it contrasts with using Forms - I think I know enough now to know practically how they're different but coming up with a concise and clear statement or paragraph about that eludes me for now.

Something like:

"You can use liferay forms for one off forms or simple use cases, but App Builder gives you flexibility to define not just the form but where people view the data, how they view the data, where they submit the form data, what the data looks like when viewing it, and where the data is presented for administrators"

What I just wrote above doesn't roll off the tongue, but it answers the question that is constantly going through my mind when I'm reading about App Builder (This sounds a lot like Forms, when and why would I use this vs Forms?)

Anyways, we don't need to solve that just to get this article up, but it's just an issue that I wanted to raise as something I don't have a great solution to yet.

### CODE REVIEW

**Comment by rbohl** [comment]
Take comfort. The struggle you're detailing is the same one we've had for a long time, since the days of DDL and Forms. Let's see if the few days of separation I've had, along with your newly written intro text, will surprisingly give birth to an explanatory epiphany in me. Otherwise I'll take you at your word,
>  we don't need to solve that just to get this article up
;)

**Comment by rbohl** [comment]
@jrhoun how is this, right after the first intro paragraph and the "Wiz-bang screenshot" I'm about to make? Too long?
------------------------
Why use app Builder when the Liferay Forms application includes almost identical form-building capability? If you're asking that question, you may not need the added features and flexibility App Builder provides. Here's when to consider using App Builder instead of Liferay Forms:

- You need more flexibility in where the app/form is deployed
- You need to display the collected data somewhere other than in the Forms application
- You need to keep the UI displaying collected data separate from other form data
- You need control over what data is displayed in the places it can be viewed
- You need multiple forms to collect data for one data object
- You need multiple data tables to display the data for one object

It's about flexibility. The ability to combine forms, data display tables, and flexible deployment options, all backed by a single data object, are what distinguish App Builder use cases from standard Forms.

**Comment by rbohl** [comment]
@jrhoun how is this, right after the first intro paragraph and the "Wiz-bang screenshot" I'm about to make? Too long?
------------------------
Why use app Builder when the Liferay Forms application includes almost identical form-building capability? If you're asking that question, you may not need the added features and flexibility App Builder provides. Here's when to consider using App Builder instead of Liferay Forms:

- You need more flexibility in where the app/form is deployed
- You need to display the collected data somewhere other than in the Forms application
- You need to keep the UI displaying collected data separate from other form data
- You need control over what data is displayed in the places it can be viewed
- You need multiple forms to collect data for one data object
- You need multiple data tables to display the data for one object

It's about flexibility. The ability to combine forms, data display tables, and flexible deployment options, all backed by a single data object, are what distinguish App Builder use cases from standard Forms.

File: docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-an-application-with-app-builder.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+Deploying an application is what makes it accessible to users. There are several ways you can make your application available to users. See the [App Builder Overview](./app-builder-overview.md#deployment) learn about the deployment options.
+
+We will deploy two applications for the Time Off Request object: one is a widget application that we can add to a site page for users to access, and the other is an administrative application in the [Product Menu](../../getting-started/navigating-dxp.md#product-menu), where the request can be managed by users with additional permissions.
+
+1. Open the Time Off Request (Control Panel &rarr; App Builder &rarr; Custom Objects).
+
+1. From the _Apps_ tab, click the Add button.
+
+    <!-- screenshot -->
```
Comment (rbohl) [inline docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-an-application-with-app-builder.md]: do we really need this screenshot here? If we have the ability to use an inline image of the add icon i could use that instead perhaps?

Comment (jrhoun) [reply docs/dxp/7.x/en/developing-applications/developing-low-code-applications/creating-an-application-with-app-builder.md]: We can try without - you can use an inline image for the add icon.

**Comment by jrhoun** [comment]
> ## @jrhoun how is this, right after the first intro paragraph and the "Wiz-bang screenshot" I'm about to make? Too long?
> Why use app Builder when the Liferay Forms application includes almost identical form-building capability? If you're asking that question, you may not need the added features and flexibility App Builder provides. Here's when to consider using App Builder instead of Liferay Forms:
> 
> * You need more flexibility in where the app/form is deployed
> * You need to display the collected data somewhere other than in the Forms application
> * You need to keep the UI displaying collected data separate from other form data
> * You need control over what data is displayed in the places it can be viewed
> * You need multiple forms to collect data for one data object
> * You need multiple data tables to display the data for one object
> 
> It's about flexibility. The ability to combine forms, data display tables, and flexible deployment options, all backed by a single data object, are what distinguish App Builder use cases from standard Forms.

Maybe we can put this into a table:

| Capabilities | Forms | App Builder |
| --- | --- | --- |
| Flexible deployment options |   | x |
| Ability to display data in response data in multiple places |   | x |
| Finer control over the display of form data  |   | x |
| Multiple forms for one data object |   | x |
| Ability to design multiple views of a single data object |   | x |

though I guess with some x's for things that are unique to forms.

**Comment by rbohl** [comment]
reworked: https://github.com/jrhoun/liferay-learn/pull/233

**Pull request closed without merging**